### ISSUE TITLE
Fails with warning about incorrect xunit async usage when not using async

### ISSUE THREAD

**Issue by JakeGinnivan** [approvals/ApprovalTests.Net#107]
I have just converted BDDfy from nUnit to xUnit 2.0 rc1 and now have a bunch of tests failing with the following error. I am not using async at all, so this is a false error. 

```
Test Name:	TestStack.BDDfy.Tests.Reporters.MarkDown.MarkDownReportBuilderTests.ShouldProduceExpectedMarkdown
Test FullName:	TestStack.BDDfy.Tests.Reporters.MarkDown.MarkDownReportBuilderTests.ShouldProduceExpectedMarkdown
Test Source:	C:\Users\Jake\_code\TestStack.BDDfy\TestStack.BDDfy.Tests\Reporters\MarkDown\MarkDownReportBuilderTests.cs : line 14
Test Outcome:	Failed
Test Duration:	0:00:00.343

Result Message:	
System.InvalidOperationException : **************************************************************************************
*                                                                                    *
* WARNING: You are using xUnit async incorrectly.
* The method MarkDownReportBuilderTests.ShouldProduceExpectedMarkdown() has a return type of void instead of Task.
* In this form this test can never fail, it will always pass.
* 
*                                                                                    *
**************************************************************************************
Result StackTrace:	
at ApprovalTests.StackTraceParsers.XUnitStackTraceParser.FailLoudly(String msg)
   at ApprovalTests.StackTraceParsers.XUnitStackTraceParser.VerifyAsyncUsedProperly(Caller res)
   at ApprovalTests.StackTraceParsers.XUnitStackTraceParser.FindApprovalFrame()
   at ApprovalTests.Namers.StackTraceParsers.AttributeStackTraceParser.Parse(StackTrace trace)
   at ApprovalTests.Namers.StackTraceParsers.StackTraceParser.Parse(StackTrace stackTrace)
   at ApprovalTests.Namers.UnitTestFrameworkNamer..ctor()
   at ApprovalTests.Approvals.<.cctor>b__c()
   at ApprovalTests.Approvals.GetDefaultNamer()
   at ApprovalTests.Approvals.Verify(IApprovalWriter writer)
   at ApprovalTests.Approvals.Verify(String text, Func`2 scrubber)
   at TestStack.BDDfy.Tests.Reporters.ReportApprover.Approve(Func`1 model, IReportBuilder reportBuilder) in C:\Users\Jake\_code\TestStack.BDDfy\TestStack.BDDfy.Tests\Reporters\ReportApprover.cs:line 18
   at TestStack.BDDfy.Tests.Reporters.MarkDown.MarkDownReportBuilderTests.ShouldProduceExpectedMarkdown() in C:\Users\Jake\_code\TestStack.BDDfy\TestStack.BDDfy.Tests\Reporters\MarkDown\MarkDownReportBuilderTests.cs:line 17
```

The code is available at https://github.com/JakeGinnivan/TestStack.BDDfy/tree/XUnitConversion

**Comment by JakeGinnivan** [comment]
Ended up downgrading that branch. https://github.com/JakeGinnivan/TestStack.BDDfy/commit/981bb4a7271c20aa58b452bb93c380701068c3e8 is the commit which is using 2.0.0

**Comment by JakeGinnivan** [comment]
Fixed in 3.0.8
